test/jvm/jvm_commands_test.rb

```diff
@@ -49,4 +49,10 @@ def test_dash_c_is_deprecated
                               )
     end
   end
+
+  def test_encoding
+    assert_output "test\n" do
+      Mirah.run('-encoding','utf-8',File.dirname(__FILE__)+"/../fixtures/utf8_test.mirah")
+    end
+  end
 end
```

Comment:
Could you add a test other than utf-8, since it'll frequently be the default? How about one that sets it to ascii and blows up on the same fixture?